### PR TITLE
readme: How it works

### PR DESCRIPTION
**Pull request by jorgeorpinel**
To review, see https://github.com/iterative/dvc/tree/readme/how-it-works#readme.

Per https://github.com/iterative/dvc/pull/7916#pullrequestreview-1011192709 (nested into that PR)

> - [x] should probably move this to AFTER the Quick start sections
> - [x] it's missing info about Experiment Management
> - [x] Extra: consistent formatting (1 sentence = 1 line)
> - [ ] Extra: better Iterative Studio mention?

* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

### CODE REVIEW

File: README.rst
Comment (jorgeorpinel) [inline README.rst:225]: I improved this a little but should we think about making a section and also adding a GIF for Studio, perhaps?

File: README.rst
Comment (jorgeorpinel) [inline README.rst:33]: Idea: put this in a <details> section (it's pretty tall, lots to scroll).

File: README.rst
Comment (shcheklein) [inline README.rst:21]: what is the purpose of this change? who many of changes are cosmetic in this PR, what should we actually review? Can we split cosmetic, things related to VS Code and other improvements? If needed do them one by one - there is no rush now.

Comment (jorgeorpinel) [reply README.rst:21]: Because of formatting, to review is best to see the the README in this branch as mentioned in the description 🙂

https://github.com/iterative/dvc/tree/readme/how-it-works#readme

There's nothing related to VS Code in this PR, it's about

> - should probably move this (**How it works**) to AFTER the **Quick start** sections
- it's missing info about Experiment Management
- Extra: consistent formatting (1 sentence = 1 line)

I can separate the last point if needed but I'm not sure what formatting we want. Is this one OK @dberenbaum ? 1 sentence = 1 line. Or should it be a fix col width? Currently it's an inconsistent mix of things.

Comment (shcheklein) [reply README.rst:21]: > I can separate the last point if needed but I'm not sure what formatting we want. Is this one OK https://github.com/orgs/iterative/teams/dvc ? (1 sentence = 1 line). Or should it be a fix col width? Currently it's an inconsistent mix of things.

yes, let's separate and do it later as a completely separate PR please

**Comment by jorgeorpinel** [comment]
This PR was due to a comment in https://github.com/iterative/dvc/pull/7916#pullrequestreview-1011192709 trying to help the core team move fwd with some old wants but it seems it's gotten complicated. I'll leave it up to you guys, feel free to reuse some of these changes. Thanks

**Comment by dberenbaum** [comment]
Thanks @jorgeorpinel! I'm finally able to get back to this. Do you want to reopen the PR, or would you prefer starting a new one?

File: README.rst
Comment (dberenbaum) [reply README.rst:21]: > * move this (**How it works**) to AFTER the **Quick start** sections
> 
> * it's missing info about Experiment Management

👍 👍 

> * Extra: consistent formatting (1 sentence = 1 line)

Not sure I understand this one. I don't think changes like the one above make any visible difference in the readme (it will still show the whole bullet as one line even with the line break), so what is the point?

**Comment by jorgeorpinel** [comment]
Should be possible to reopen and just take over the branch, if that's helpful! We could also split it up as well and I can focus on getting part of it merged (please lmk which part I should isolate if so). Thanks @dberenbaum

**Comment by dberenbaum** [comment]
@jorgeorpinel Could you take a look at https://github.com/iterative/dvc/pull/7917#discussion_r912108647? That's the only thing that's unclear and blocking the PR from being merged IMO.

File: README.rst
Comment (jorgeorpinel) [reply README.rst:21]: > I don't think changes like the one above make any visible difference in the readme ... so what is the point?

Just markup code consistency. Currently there's a mix of lines wrapped at different column lengths and some that don't seem to have any limit.

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
@jorgeorpinel Do you mind rebasing?

**Comment by jorgeorpinel** [comment]
Oops. I deleted the base branch 😅 (since it doesn't have any changes on top of `main`). Let me reopen this correctly... ⏳

**Pull request closed without merging**
